src/build-deps.txt

```diff
@@ -2,3 +2,5 @@
 time
 # Used by mantle
 golang
+# Use tmux for developer multi-arch builder access UX
```

Comment:
maybe add a link to https://github.com/coreos/fedora-coreos-pipeline/issues/803 here

Reply:
ok, I have added the link in the comment